modules/loop_notification/loop_notification.install

```diff
@@ -10,12 +10,9 @@
 function loop_notification_update_7001() {
```

Comment:
Are we allowed to change an existing update hook? Shouldn't we create a new one to clean up?

And shouldn't this be part of the previous PR (https://github.com/os2loop/profile/pull/113)?

Reply:
Well either the update has already run in which case it will never be run again or it has not yet run and it might fail since the content types it's acting on doesn't exist, so i think this is right.